tests/run_examples.py

```diff
@@ -25,6 +26,10 @@
 from daal4py import __has_dist__
 from daal4py.sklearn._utils import get_daal_version
 
+parser = argparse.ArgumentParser()
+parser.add_argument("--assert_gpu", action="store_true")
```

Comment:
I think it's more typical to use only hyphens in CLI arguments, i.e. `--assert-gpu`. The variable is still available as `args.assert_gpu`, so you can rename it here without further changes.

```suggestion
parser.add_argument("--assert-gpu", action="store_true")
```